simulator/generation/property.rs

```diff
@@ -164,10 +183,43 @@ impl Property {
 
                 interactions
             }
+            Property::SelectLimit { select } => {
+                let table_name = select.table.clone();
+
+                let assumption = Interaction::Assumption(Assertion {
+                    message: format!("table {} exists", table_name),
+                    func: Box::new({
+                        let table_name = table_name.clone();
+                        move |_: &Vec<ResultSet>, env: &SimulatorEnv| {
+                            Ok(env.tables.iter().any(|t| t.name == table_name))
+                        }
+                    }),
+                });
+
+                let limit = select.limit.unwrap_or(0);
```

Comment:
I assume the `Select` inside `SelectLimit` should always have a nonzero limit so should this just be `.expect("Property::SelectLimit without a LIMIT clause")` ?

Reply:
That makes sense 👍🏻